docs/en_US/sdk_reference.rst

```diff
@@ -7,4 +7,5 @@ Python API Reference
     :maxdepth: 1
 
     Auto Tune <autotune_ref>
-    NAS <NAS/NasReference>
\ No newline at end of file
+    NAS <NAS/NasReference>
+    CompressionUtils <Compressor/CompressionUtils>
```

Comment:
this should be reference not tutorial, could follow `autotune_ref` and `NAS/NasReference`

Reply:
I have updated the docs, thanks~